### PR TITLE
Use eurlatgr font instead of lat9w-16

### PR DESCRIPTION
**Pull request by imobachgs**
Replaces `lat9w-16` with `eurlatgr`.

### CODE REVIEW

**Comment by Vogtinator** [comment]
Isn't it possible to just merge consolefonts with consolefonts_opensuse.ycp?

**Comment by imobachgs** [comment]
@Vogtinator Yes, we could merge both of them. But we would need to fix some conflicts. Basically:

* If you check `de_DE`, `de_DE`, `de_CH` and `de` for SLE, all of them are using `lat1-16.psfu`. We would keep them, but `de` is also used as fallback in the openSUSE file, so we have a conflict here (not only the font, but the unicode map, screen map and console magic parameters).

On the other hand, should we keep entries like `en_US.UTF-8`? openSUSE would fallback to `en`, which contains a different set of values for unicode map, screen map and console magic.

Opinions?

**Comment by Vogtinator** [comment]
IMO all of those should use eurlatgr, like the opensuse map says.
Then you shouldn't need any unicode or screen maps anymore.

**Comment by imobachgs** [comment]
Ok, then switching to use the openSUSE map, as it is, should do the trick. I only have spotted one conflict using this approach: `tr` used `iso09.f16n` in SLES but `eurlatgr` in openSUSE. I guess it is safe to replace it with `eurlatgr`.

**Comment by Vogtinator** [comment]
Looks good to me - I searched for `iso09.f16n` everywhere but the only hits were in YaST or related parts so AFAICT it should be safe to not use it for `tr`.

**Comment by imobachgs** [comment]
@Vogtinator Thanks. I will merge as soon as Travis is happy again.

File: package/yast2-country.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jan 24 13:39:34 UTC 2018 - igonzalezsosa@suse.com
+
+- Use eurlatgr font instead of lat9w-16 (bsc#1068814).
```
Comment (jreidinger) [inline package/yast2-country.changes]: maybe also mention that with this change we sync opensuse and SLE

**Pull request merged**